docs/resources/task.md

```diff
@@ -63,6 +64,7 @@ resource "iterative_task" "example" {
 - `parallelism` - (Optional) Number of machines to be launched in parallel.
 - `storage.workdir` - (Optional) Local working directory to upload and use as the `script` working directory.
 - `storage.output` - (Optional) Results directory (**relative to `workdir`**) to download (default: no download).
+- `storage.exclude` - (Optional) List of files and globs to exclude from transfering. Excluded files are neither uploaded to cloud storage nor downloaded from it. Exclusions are defined relative to `storage.workdir`.
```

Comment:
globs conforming to which spec?

Reply:
Globs like in things-other-than-*-are-implementation-dependent-and-this-is-not-a-glob

Reply:
https://rclone.org/filtering 😉